spec/rails_helper.rb

```diff
@@ -12,6 +12,8 @@
 require 'capybara/rails'
 require 'capybara/rspec/matchers'
 
+Capybara.server = :puma, { Silent: true }
```

Comment:
I don't think we need this anymore now that we're using `govuk_test`.

Reply:
We do if we want to silence Puma logs and prevent it from writing
```
Puma starting in single mode...
* Version 3.10.0 (ruby 2.4.1-p111), codename: Russell's Teapot
* Min threads: 0, max threads: 1
* Environment: test
* Listening on tcp://0.0.0.0:36775
Use Ctrl-C to stop
```
while running the test suite. See https://github.com/rspec/rspec-rails/issues/1897

Reply:
...unless we modify `govuk_test` and put this directly there.

Reply:
Ah I see! Yeah, I think that sounds like something worth adding to `govuk_test`.

Reply:
I'll remove it from here and I'll add it to `govuk_test`

Reply:
https://github.com/alphagov/govuk_test/pull/16

Reply:
https://github.com/alphagov/govuk_test/pull/17